set.mm

```diff
@@ -15569,42 +15569,35 @@ only postulated (that is, axiomatic) rule of inference of predicate
   The empty logical domain
 -.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
 
-  The empty logical domain is certainly a limit case, that is in contradiction
-  with set theory because ~ ax-nul postulates the existence of at least one
-  set, ~ ax-pr postulates the existence of at least two sets, and finally
-  ~ ax-inf postulates the existence of an infinite number of sets. It is
-  nevertheless interesting to shortly develop this case because the results are
-  exactly meeting our intuition. Every universally quantified wff is provably
-  true, and every existentially quantified wff is provably false. Otherwise,
-  nothing can be said. Reciprocally, in the case that every universally
-  quantified wff is true, then ~ ax-6 is negated, but ~ ax-gen , ~ ax-4 ,
-  ~ ax-5 , ~ ax-10 , ~ ax-11 , ~ ax-12 and ~ ax-13 are automatically true, and
-  so are also every universal quantification of ~ ax-7 , ~ ax-8 and ~ ax-9 .
-$)
-
-  $( In an empty logical domain existential quantification never holds, not
-     even for false expressions.  (Contributed by Wolf Lammen, 21-Jan-2024.) $)
-  nax6ex $p |- ( -. E. x T. -> -. E. x ph ) $=
+  This database develops mathematics from first-order logic, which has only
+  nonempty models.  Before stating axioms excluding the empty model
+  (typically, ~ ax-6 in logic and ~ ax-nul in set theory), we state in this
+  short subsection a few results relative to the empty domain, which we
+  characterize by the assumption ` -. E. x T. ` .  As expected, on the empty
+  domain, every universally quantified formula is true ( ~ emptyal ) and every
+  existential formula is false (~ emptyex ), and every variable is effectively
+  nonfree in any formula ( ~ emptynf ).
+$)
```

Comment:
I cannot "suggest" a modification here (maybe because it's a draft), but there should be a blank line right before `$)`.